rest-mvc/src/main/java/com/intuit/tank/rest/mvc/rest/controllers/AgentController.java

```diff
@@ -11,7 +11,7 @@
 import com.intuit.tank.vm.agent.messages.AgentTestStartData;
 import com.intuit.tank.vm.agent.messages.AgentData;
 import com.intuit.tank.api.model.v1.cloud.CloudVmStatus;
```

Comment:
@Zakaria-Kofiro  Should we be referencing https://github.com/intuit/Tank/blob/zkofiro/tank-api-V2/rest-mvc/src/main/java/com/intuit/tank/rest/mvc/rest/models/agent/CloudVmStatus.java?

Reply:
Fixed: Import is referencing the right data model, it's returned by V1 CloudController which Agent Service still depends on for agent instance operations. Replicating JobController and CloudController caused a lot of issues in job/agent instance operations, so they are the only V1 references left, and will be moved over as V1 is sunset. 

Removed the unused data models for now and will add them back in when removing V1.